lamindb/curators/__init__.py

```diff
@@ -2171,7 +2159,7 @@ def _add_set_from_spatialdata(
                 """Add Schemas from SpatialData."""
                 if obs_fields is None:
                     obs_fields = {}
-                assert host.otype == "spatialdata"  # noqa: S101
+                assert host.otype == "SpatialData"  # noqa: S101
```

Comment:
If we're changing the convention here we need to run a data migration, too. Otherwise this here will be a mess:

<img width="516" alt="image" src="https://github.com/user-attachments/assets/51a0bf3a-8f80-4adb-acc0-2df1a558b49d" />

Can of course deal with this manually, but that's not nice.

Hmmm. I'm running another data migration anyway this weekend (PR that got merged to main), and could take care of this.

But this means we have to merge this PR and have it be part of lamindb 1.2.

The PR looks great up to the uncovered lines. I assume the only thing that misses are some tests? Is that true?

Reply:
Yup, it's just super minor things. I wanted to get this PR ready Monday morning so let me do that and then we can merge it with 1.2